pytest/tests/mocknet/helpers/load_test_spoon_helper.py

```diff
@@ -171,100 +55,56 @@ def get_test_accounts_from_args(argv):
                         rpc_infos=rpc_infos) for key in test_account_keys
     ]
     max_tps_per_node = max_tps / num_nodes
-    return node_account, accounts, max_tps_per_node
-
-
-def init_ft(node_account):
-    tx_res = node_account.send_deploy_contract_tx(
-        '/home/ubuntu/fungible_token.wasm')
-    logger.info(f'ft deployment {tx_res}')
-    mocknet_helpers.wait_at_least_one_block()
-
-    s = f'{{"owner_id": "{node_account.key.account_id}", "total_supply": "{10**33}"}}'
-    tx_res = mocknet_helpers.retry_and_ignore_errors(
-        lambda: node_account.send_call_contract_raw_tx(
-            node_account.key.account_id, 'new_default_meta', s.encode('utf-8'),
-            0))
-    logger.info(f'ft new_default_meta {tx_res}')
-
-
-def init_ft_account(node_account, account):
-    s = f'{{"account_id": "{account.key.account_id}"}}'
-    tx_res = mocknet_helpers.retry_and_ignore_errors(
-        lambda: account.send_call_contract_raw_tx(
-            node_account.key.account_id, 'storage_deposit', s.encode('utf-8'),
-            (10**24) // 800))
-    logger.info(f'Account {account.key.account_id} storage_deposit {tx_res}')
-
-    # The next transaction depends on the previous transaction succeeded.
-    # Sleeping for 1 second is the poor man's solution for waiting for that transaction to succeed.
-    # This works because the contracts are being deployed slow enough to keep block production above 1 bps.
-    mocknet_helpers.wait_at_least_one_block()
-
-    s = f'{{"receiver_id": "{account.key.account_id}", "amount": "{10**18}"}}'
-    logger.info(
-        f'Calling function "ft_transfer" with arguments {s} on account {account.key.account_id}'
-    )
-    tx_res = mocknet_helpers.retry_and_ignore_errors(
-        lambda: node_account.send_call_contract_raw_tx(
-            node_account.key.account_id, 'ft_transfer', s.encode('utf-8'), 1))
-    logger.info(
-        f'{node_account.key.account_id} ft_transfer to {account.key.account_id} {tx_res}'
-    )
+    return load_test_utils.TestState(node_account, accounts, max_tps_per_node,
+                                     rpc_infos)
```

Comment:
Personally I much prefer the 'one arg one line' formatting. You can achieve that by adding a trailing comma after the last arg. It's personal preference so up to you ;)

Reply:
Looks good, thank you